packages/upstream/tar.2.3.0/opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "tar"
-version: "2.0.1"
+version: "2.3.0"
```

Comment:
Would be useful to add a note on how the new tar 2.5.1 packages are buggy (e.g. links to issues/PRs) so we can check whether they are fixed in the future.

Reply:
Hmm, I think the problematic version was 2.5.0, and the fix was released in 2.5.1